server/models/Card.js

```diff
@@ -10,6 +10,26 @@ const CardSchema = new Schema(
     description: {
       type: String,
     },
+    color: {
+      type: String,
+    },
+    deadline: {
+      type: String,
+    },
+    tags: [
+      {
+        name: {
+          type: String,
+        },
+      },
+    ],
+    attachments: [
+      {
+        imgSrc: {
```

Comment:
Let's call this src for now, because eventually we may want to include attachments that aren't images